styles/globals.css

```diff
@@ -3,3 +3,7 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+body {
+    background-color: #1F2328;
```

Comment:
Why are we changing the backgorund-color?

Reply:
The background color given to the divs was not covering the whole body so if we have a big page then it will show the background color for only the window size and will show white background for rest of the page. I have reverted this change as Roberto will be changing the style changes

![style](https://user-images.githubusercontent.com/85132632/191724368-89b8880f-967e-487a-b6b2-7992c4ad2343.PNG)